README.md

```diff
@@ -88,52 +88,110 @@ curl -L https://aka.ms/install-dmr-client-linux | bash
+
+> dmr-client import --model-file "MyThermostat.json"
 ```
 
->Note: You can use the `--local-repo` argument to specify the local repo root folder
+> This command will rename and locate the file in the appropriate folder
 
 ### Validate Models
 
 You can validate your models with the `dmr-client validate` command.
 
 ```bash
-dmr-client validate --model-file ./my/model/file.json
+# Validate a model file using the DTDL parser
+
+> dmr-client validate --model-file ./my/model/file.json
 ```
 
 >Note: The validation uses the latest DTDL parser version to ensure all the interfaces are compatible with the DTDL language spec
 
 To validate external dependencies, those must exist in the local repo. To validate those you can specify a `local` or `remote` folder to validate against.
 
 ```bash
-# from the repo root folder
-dmr-client validate --model-file ./my/model/file.json --repo .
+# Validate a model file using the DTDL parser checking dependencies with the current folder as a local repo
+
+> dmr-client validate --model-file ./my/model/file.json --repo .
 ```
 
 #### Strict validation
 
 The Device Model Repo includes additional [requirements](pr-reqs.md), these can be validated with the `strict` flag.
 
 ```bash
-dmr-client validate --model-file ./my/model/file.json --repo . --strict true
+# Validate a model file using the DTDL parser checking dependencies with the current folder as a local repo in strict mode
+
+> dmr-client validate --model-file ./my/model/file.json --repo . --strict true
 ```
 
 #### Export models
 
 Models can be exported from a given repo (local or remote) to a single file using a JSON Array. 
 
 ```bash
-dmr-client export --dtmi "dtmi:com:example:TemperatureController;1" -o TemperatureController.expanded.json
+# Retrieves an interface from a custom repo by DTMI
+
+> dmr-client export --dtmi "dtmi:com:example:Thermostat;1" --repo https://raw.githubusercontent.com/Azure/iot-plugandplay-models/main
+```
+
+#### Create Index
+
+The model repo can host a `index.json` file with all the `ids` avaialble in the repository. Read the [Index Spec](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Model-Index)
+
+```bash
+# Builds a model index for the repository. If models exceed the page limit new page files will be created relative to the root index.
+
+> dmr-client index --local-repo .
+```
+
+```bash
+# Build a model index with a custom page limit indicating max models per page.
+
+> dmr-client index --local-repo . --page-limit 100
+```
+
+#### Create Expanded files
+
+```bash
+# Expand all models from the root directory of a local models repository following Azure IoT conventions.
+# Expanded models are inserted in-place.
+
+> dmr-client expand --local-repo .
+```
+
+```bash
+# The default --local-repo value is the current directory. Be sure to specifiy the root for --local-repo.
+
+> dmr-client expand
 ```
 
 ## Consuming
 
+The *Models Repository Client* libraries are available in the Azure SDK for the next platforms:
```

Comment:
This sentence needs to be tweaked a bit.  Maybe something like

"There are Azure SDKs available for the models repository in the following languages: "